src/views/mfa-verify/PushForm.js

```diff
@@ -50,6 +50,15 @@ function (Okta, CookieUtil, Util, NumberChallengeView) {
         }
       );
 
+      this.listenTo(this.options.appState, 'change:isMfaRejectedDueToOutdatedApp',
+        function (state, isMfaRejectedDueToOutdatedApp) {
```

Comment:
it would be clean to create a separate new handler for this.

Reply:
You mean not use a  inline function ?

Reply:
yeah.

Reply:
@pradeepdewda-okta  not sure if we are adding much value here since all other functions are inline and we are not unit testing this function per say(SIW unite tests are more flow based). 

Also naming these functions are tricky.